docs/tools/vdb_table/data/usearch.json

```diff
@@ -25,26 +25,26 @@
     "source_url": "",
     "comment": ""
   },
-  "github_stars": 1061,
+  "github_stars": 1250,
   "vector_launch_year": 2023,
   "metadata_filter": {
-    "support": "",
+    "support": "partial",
     "source_url": "",
-    "comment": ""
+    "comment": "hybrid filtering techniques are available in some SDKs, like C++, but not in Python"
   },
   "hybrid_search": {
-    "support": "",
+    "support": "partial",
     "source_url": "",
-    "comment": ""
+    "comment": "hybrid filtering techniques are available in some SDKs, like C++, but not in Python"
```

Comment:
please provide a reference to an example where Hybrid Search is achieved within the DB (i.e. the user is able to merge keyword and vector based rankings into a single ranked list).

Reply:
[Docs](https://unum-cloud.github.io/usearch/cpp/reference.html#_CPPv4I0000ENK4unum7usearch8index_gt6searchE15search_result_tRR8value_atNSt6size_tERR9metric_at21index_search_config_tRR12predicate_atRR11prefetch_at). [Code](https://github.com/unum-cloud/usearch/blob/6b43d1ce39e0381016e121981615657af86e9568/include/usearch/index.hpp#L2447-L2462).

Reply:
The links seem to show the ability to do filtered vector search which is a different concept from what this column is meant to represent. Moving this back to blank